web/styles/compose.css

```diff
@@ -1270,8 +1270,8 @@ textarea.new_message_textarea {
         margin: 0 5px;
         /* Coordinate with the value of
            --compose-formatting-buttons-row-height */
-        /* 22px at 14px/1em */
-        height: 1.5714em;
+        /* 16px at 14px/1em */
+        height: 1.1429em;
```

Comment:
This seems to be in the wrong commit.

Reply:
Yeah. I've rebased and put it in its own commit.